shell/utils/time.ts

```diff
@@ -68,7 +71,7 @@ export function getSecondsDiff(startDate, endDate) {
  * diff:  update frequency in seconds
  * label: content of the cell's column
  */
-export function elapsedTime(seconds) {
+export function elapsedTime(seconds: any) {
```

Comment:
nit: where possible remove the `any` type

Reply:
This out of the scope and we do not have TS refinements.